pkgs/applications/misc/sequeler/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchFromGitHub,
+  cmake, pkgconfig,
+  gtk3, granite, libgee, libxml2, gtksourceview, libgda, vala }:
+
+let
+  version = "0.5.3";
+  sqlGda = libgda.override {
+    mysqlSupport = true;
+    postgresSupport = true;
+  };
+
+in stdenv.mkDerivation rec {
+  name = "sequeler-${version}";
+
+  src = fetchFromGitHub {
+    owner = "Alecaddd";
+    repo = "sequeler";
+    rev = "v${version}";
+    sha256 = "0m5zwl9jfdl1dzd1ymlwx7rx5cr9fdx06sbnidaajh33z02zaph0";
+  };
+
+  nativeBuildInputs = [ cmake pkgconfig vala ];
```

Comment:
`ninja` is preferred with `cmake`.

Reply:
It doesn't seem to enjoy that.

```
ninja: error: loading 'build.ninja': No such file or directory
```

Reply:
It is depends on the order, I think. See my package above.

Reply:
I've tried to put cmake before or after ninja in that list, no difference :/

Reply:
Oh, right. The problem is the manual `configurePhase`. That should not be needed. Again, see https://github.com/NixOS/nixpkgs/pull/35472#issuecomment-368235256

Reply:
Ah, missed that. Seems to build fine without it. Added ninja as well and amended my commit :)